wp-parsely.php

```diff
@@ -104,12 +104,15 @@ public function __construct() {
 		add_action( 'save_post', array( $this, 'update_metadata_endpoint' ) );
 		add_action( 'instant_articles_compat_registry_analytics', array( $this, 'insert_parsely_tracking_fbia' ) );
 		add_action( 'template_redirect', array( $this, 'parsely_add_amp_actions' ) );
-		if ( ! defined( 'WP_PARSELY_TESTING' ) ) {
+		if ( ! self::is_test_env() ) {
```

Comment:
Outside of the scope of this PR, but this really shouldn't be needed! Need to see why tests supposedly break when some styles are enqueued!

Reply:
Opened https://github.com/Parsely/wp-parsely/issues/256.